app.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topicsController");
+
+
+const app = express();
+app.use(express.json());
+
+app.get('/api/topics', getTopics)
+
+app.use((err, req, res, next) => {
```

Comment:
You aren't using this PSQL error handler for this, so it shouldn't be added until you get to a ticket where you need it